### PR TITLE
New projects built on Oasis - Update oasis.toml

### PR DESCRIPTION
**Pull request by MihneaOasis**

### CODE REVIEW

File: data/ecosystems/o/oasis.toml [hunk 1]
```diff
@@ -541,3 +541,114 @@ url = "https://github.com/viraja1/privacy_compute_platform"
 
 [[repo]]
 url = "https://github.com/zoek1/FastGoverment"
+
+[[repo]]
+url = "https://github.com/therealbryanho/super-chad-adventures/blob/main/README.md"
```
Comment (jubos) [inline data/ecosystems/o/oasis.toml:546]: These urls need to be the url of the root gitrepo: https://github.com/therealbryanho/super-chad-adventures

**Review by jubos — CHANGES_REQUESTED** [review]
Please standardize the URL formats.

**Review by MihneaOasis — COMMENTED** [review]
FIle updated

**Comment by jubos** [comment]
This still needs some improvements before it can be merged.  There are a lot of Github users in the repository section and still a few direct links to files instead of the repos.

**Review by jubos — CHANGES_REQUESTED** [review]
This still needs some improvements before it can be merged. There are a lot of Github users in the repository section and still a few direct links to files instead of the repos.

**Comment by jubos** [comment]
Going to close this out for now.  Please reopen when it is ready for review after updating for the feedback.

**Pull request closed without merging**